pyproject.toml

```diff
@@ -33,7 +33,6 @@ warn_unreachable = true
 pretty = true
 show_column_numbers = true
 show_error_context = true
-show_error_codes = true
```

Comment:
?

Reply:
Oops, yeah, this is default behaviour for a couple years now. The option isn't documented anymore.

Reply:
No harm in leaving it to be explicit though, yeah?

Reply:
Normally I wouldn't care, but the option literally hasn't existed in mypy for 2 years: https://github.com/python/mypy/pull/13542

So, that is not actually doing anything - not even re-setting a default value. Just ignored.